frontend/src/utils/accounts.ts

```diff
@@ -0,0 +1,10 @@
+import { generatePrivateKey, privateKeyToAccount } from 'viem/accounts'
+
+export const getAccount = (privateKey: `0x${string}`) => {
+    return privateKeyToAccount(privateKey)
+}
+
+export const getPrivateKey = () => {
```

Comment:
@bradleySuira should these wrappers go on "utils"?
generate looks like a better name for this one and getAccountFromPrivatekey also looks better in my opinion for the other method

Reply:
for now in utils but we can define later if we need more related to wallet restore and similar the name is now your suggestion your are right